### PR TITLE
ADMIN PR - Updates to topologies/kne files

### PR DESCRIPTION
**Pull request by octpetre**
- Changed otg version to 9999. Users will have to adjust the otg version based on their release found [here](https://github.com/open-traffic-generator/ixia-c/releases)
- Added _juniper_ixia.textproto_ and _juniper_ixia_lag.textproto_
- Updated _juniper.config_

### CODE REVIEW

File: topologies/kne/arista_ixia.textproto [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: "arista-ixia"
 nodes: {
     name: "otg"
     vendor: KEYSIGHT
-    version: "0.0.1-3423"
+    version: "0.0.1-9999"
```
Comment (liulk) [inline topologies/kne/arista_ixia.textproto:5]: As discussed, please add some explanation (possibly a README.md in topologies/kne) about how to determine the correct version number to use.

Let's put a comment here also to indicate that this is a placeholder.

```suggestion
    # Please update this with the local version number of the foo component.
    version: "TODO"
```

Comment (octpetre) [reply topologies/kne/arista_ixia.textproto:5]: Added topologies/kne/README.md file

File: topologies/kne/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+The otg version must be set according to the available release found [here](https://github.com/open-traffic-generator/ixia-c/releases)
```
Comment (liulk) [inline topologies/kne/README.md:1]: Can you expand the README to put the version setting in context of a short guide what needs to be done the first time someone tries to run an OTG test?

In particular:

 1. What are the prerequisites? Instructions for running Ixia-C or links to instructions?
 2. What are the files in this directory for?
 3. What changes need to be made locally? Be specific: each release has a versions.yaml file. Get the version from the "release" key and use it to update the version placeholder in the *.textproto file.
 4. Example command line.

File: topologies/kne/arista_ixia.textproto (hunk 1 above)
Comment (liulk) [reply topologies/kne/arista_ixia.textproto:5]: Please add a comment to make the placeholder explicit.

**Review by liulk — CHANGES_REQUESTED** [review]

File: topologies/kne/README.md (hunk 2 above)
Comment (octpetre) [reply topologies/kne/README.md]: Updated README.md file

**Review by liulk — APPROVED** [review]

**Pull request merged**